liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -1092,7 +1090,8 @@ protected void doExecute() throws Exception {
         } else {
             runLibertyMojoCreate();
             if (generateFeatures) {
-                runLibertyMojoGenerateFeatures(null);
+                // generate features on startup - provide all classes and only user specified features to binary scanner
+                runLibertyMojoGenerateFeatures(null, Boolean.valueOf(true));
```

Comment:
If we're sticking with Boolean this could be Boolean.TRUE. But the compiler would probably optimize it anyway.